array/plus_one.py

```diff
@@ -36,3 +36,13 @@ def plus_one(digits):
     new_num = [0] * (n+1)
     new_num[0] = 1
     return new_num
+## another implementation, convert list to string, then to integer, then increment that integer
+def plusone(digits):
+        num=""
+        for d in digits:
+            num+=str(d)
+        incremented=str(int(num)+1)
+        incrementedAsList=[]
```

Comment:
Indeed, but it would convert it to a list of chars, while the needed return type was a list of ints, that's why I used a loop.